test/test.js

```diff
@@ -577,4 +617,17 @@ describe('finalhandler(req, res)', function () {
         })
     })
   })
+
+  describe('should display cause in the Error output', function () {
+    it('should return error message with stack trace in development', function (done) {
+      process.env.NODE_ENV = 'development'
+      const err = new Error('foo', { cause: new Error('bar') })
+
+      const server = createServer(err)
+
+      request(server)
+        .get('/')
+        .expect(500, /[Error: bar]/, done)
```

Comment:
May want to modify this test case a bit as it does not confirm the `cause` trace is printed anywhere.

Reply:
I wrote a similar test case here if you want to use it https://github.com/coltonehrman/bag-of-tools/blob/main/examples/express/error-stack-trace/error.test.ts

Reply:
I think you addressed this in your new PR, and since we haven't heard back from @DanielBelz1997 on reverting the formatting changes, I am considering closing this in favor of #50. @DanielBelz1997 opened theirs first, I want to give them a chance to fix this PR. I have some feedback I will leave on #50 as well, and if those are addressed first we will move forward with #50 instead of this one.